src/fragments/lib/auth/flutter/getting_started/60_developerPreview.mdx

```diff
@@ -1,6 +1,6 @@
 <Callout warning>
 
-Web and desktop support for the Amplify Flutter Auth category is now in developer preview!  See the documentation for [project setup](/lib/project-setup/create-application) for more information about getting started.
+Web and desktop support for the Amplify Flutter Storage category is now in developer preview!  See the documentation for [project setup](/lib/project-setup/create-application) for more information about getting started.
```

Comment:
Should this list all the categories?

This is in the Auth library's homepage, so I think it would make sense to only callout Auth here.

Reply:
There are other places with this callout, too, like /start/q/integration/flutter/. Should we update all locations?

Reply:
Oops I think I committed wrong changes. Will update